portfolio/src/main/webapp/index.html

```diff
@@ -82,5 +81,79 @@ <h2 class="subtitle is-3" id="greet">
       }, 2000);
     </script>
 
+    <section class="section">
+      <div class="level">
+        <div class="level-item content">
+          <h1>Leave me a message!</h1>
+        </div>
+      </div>
+      <div class="level">
+        <div class="level-item">
+          <form action="/data" method="post" id="myform">
+
+            <div class="field">
+              <label class="label">Name</label>
+              <div class="control has-icons-left has-icons-right">
+                <input class="input is-info" type="text" placeholder="Full Name" name="name" required="required">
+                <span class="icon is-small is-left">
+                  <i class="fas fa-user"></i>
+                </span>
+              </div>
+            </div>
+
+            <div class="field">
+              <label class="label">Email</label>
+              <div class="control has-icons-left has-icons-right">
+                <input class="input is-info" type="email" placeholder="email@example.com" name="email"required="required">
+                <span class="icon is-small is-left">
+                  <i class="fas fa-envelope"></i>
+                </span>
+              </div>
+            </div>
+
+            <div class="field">
+              <label class="label">Message</label>
+              <div class="control">
+                <textarea class="textarea is-info" placeholder="Message for Cicilia..." name="message" required="required"></textarea>
+              </div>
+            </div>
+
+            <div class="field is-grouped">
+              <div class="control">
+                <button class="button is-link">Submit</button>
+              </div>
+              <div>
+                <button class="button is-link is-light" type="button" onclick="clearAll()">Clear</button>
+              </div>
+            </div>
+
+          </form>
+        </div>
+      </div>
+
+      <script>
+        function clearAll() {
+          document.forms.myform.elements.name.value = null
+          document.forms.myform.elements.email.value = null
+          document.forms.myform.elements.message.value = null
```

Comment:
Copying from previous comment: It's preferable to use null sparingly, because it has unexpected behaviour in many cases. Maybe you can just use an empty string here?

Reply:
ok